agent/cita_exporter/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM ubuntu:18.04
+FROM rust:1.48 AS builder
+WORKDIR /build
+RUN curl -o musl-1.1.19.tar.gz https://www.musl-libc.org/releases/musl-1.1.19.tar.gz \
+	&& tar -xzvf musl-1.1.19.tar.gz \
+	&& cd musl-1.1.19/ \
+	&& ./configure && make && make install \
+	&& ln -sf /usr/local/musl/bin/musl-gcc /usr/local/bin/musl-gcc
+COPY ./cita-cli.git /build/cita-cli.git
+WORKDIR /build/cita-cli.git/cita-cli
+RUN rustup target add x86_64-unknown-linux-musl \
+	&& RUSTFLAGS='-A deprecated' cargo install --target x86_64-unknown-linux-musl --path .
+
+FROM ubuntu:18.04 AS final
```

Comment:
should add a comment for what is this sage used for, for example: `# install requirements for cita_monitor_agent.py`